ACPCampaignClassic.podspec

```diff
@@ -1,27 +1,60 @@
 Pod::Spec.new do |s|
   s.name         = "ACPCampaignClassic"
-  s.version      = "1.1.0"
-  s.summary      = "Campaign Classic framework for Adobe Experience Platform SDK. Written and Supported by Adobe."
+  s.version      = "2.0.0"
+  s.summary      = "Campaign Classic framework for Adobe Experience Cloud SDK. Written and maintained by Adobe."
   s.description  = <<-DESC
-                   The campaign classic framework provides APIs that allow use of the Campaign Classic product in the V5 Adobe Experience Platform SDK.
+                   The campaign classic framework provides APIs that allow use of the Campaign Classic product in the V5 Adobe Experience Cloud SDK.
                    DESC
 
   s.homepage     = "https://github.com/Adobe-Marketing-Cloud/acp-sdks/releases"
 
-  s.license      = {:type => "Commercial", :file => "LICENSE.md"}
-  s.author       = "Adobe Experience Platform SDK Team"
-  s.source       = { :git => 'https://github.com/Adobe-Marketing-Cloud/acp-sdks.git', :tag => "v1.1.0-ACPCampaignClassic" }
-  s.platform     = :ios, '10.0'
-
-  # dependency on the core framework
-  s.dependency "ACPCore", "~> 1.0"
-
-  s.subspec 'iOS' do |ios|
-    ios.vendored_frameworks = 'ACPCampaignClassic_iOS.framework'
-    ios.xcconfig = {
-      "FRAMEWORK_SEARCH_PATHS" => "'${PODS_ROOT}/ACPCampaignClassic'",
-      "LD_RUNPATH_SEARCH_PATHS" => "@loader_path/../Frameworks"
-    }
-    ios.requires_arc = true
+  s.license      = {:type => "Commercial", :text => "Adobe.  All Rights Reserved."}
+  s.author       = "Adobe Mobile SDK Team"
+  s.source       = { :git => 'git@git.corp.adobe.com:dms-mobile/v5-ios-builds.git', :tag => "v#{s.version}-#{s.name}" }
+  s.platform = :ios, "10.0"
+  s.requires_arc = true
+
+  s.default_subspec = "iOS"
+  s.static_framework = true
+
+  s.dependency "ACPCore", "~> 2.0"
+
+  s.subspec "iOS" do |ios|
+    ios.vendored_libraries = "iOS/libACPCampaignClassic_iOS.a"
+    ios.source_files = "iOS/include/*.h", "iOS/include/*.m"
+    ios.frameworks = "UIKit", "SystemConfiguration"
```

Comment:
Both the `ios.frameworks` and `ios.libraries` lines can be removed. These dependencies are covered in the ACPCore podspec.